.clang-format

```diff
@@ -26,7 +26,7 @@ IndentWidth: 2
 ObjCBlockIndentWidth: 2
 ObjCSpaceAfterProperty: true
 ObjCSpaceBeforeProtocolList: true
-PointerAlignment: Left
+PointerAlignment: Right
```

Comment:
Just gonna sneak this in here.

Reply:
(It's Apple's convention and `Left` wasn't getting applied consistently by clang-format anyways.)

Reply:
No opinion here, but for context it was `Left` because that's what upstream Envoy uses for C++